test/ng/directive/booleanAttrsSpec.js

```diff
@@ -283,8 +284,9 @@ describe('ngHref', function() {
     expect(element.attr('href')).toEqual(undefined);
   }));
 
-  if (msie) {
-    // IE11/10/Edge fail when setting a href to a URL containing a % that isn't a valid escape sequence
+  // Support: IE 9-11 only, Edge 12-14+
+  if (msie || /\bEdge\/[\d\.]+\b/) {
```

Comment:
Ah, no, this particular condition was necessary, the tests are now failing. Fixing...